integration-cli/daemon/daemon_swarm.go

```diff
@@ -96,6 +96,18 @@ func (d *Swarm) SwarmInfo() (swarm.Info, error) {
 	return info.Swarm, nil
 }
 
+// Unlock tries to unlock a locked swarm
+func (d *Swarm) Unlock(req swarm.UnlockRequest) error {
+	status, out, err := d.SockRequest("POST", "/swarm/unlock", req)
+	if status != http.StatusOK {
+		return fmt.Errorf("unlocking swarm: invalid statuscode %v, %q", status, out)
+	}
+	if err != nil {
+		err = fmt.Errorf("unlocking swarm: %v", err)
```

Comment:
import `github.com/pkg/errors` and use `err = errors.Wrap(err, "unlocking swarm")`